bertytech/content/docs/community/index.md

```diff
@@ -138,13 +138,12 @@ There are lots of ways to help on the creative side. We can channel our creative
 
 ### ㊗️ Translation
 
-As you may guess, we don't know every language. However, we want Berty to be accessible to as many people as possible. That's why we need as much material as possible translated into as many languages as possible. 
+As you may guess, we don't know every language. However, we want Berty to be accessible to as many people as possible. That's why we need as much material as possible translated into as many languages as possible.
 
-This can be the documentation, the app, the support, a blog post, even this page, or another section on the site. 
+This can be the documentation, the app, the support, a blog post, even this page, or another section on the site.
 
-➡️ https://translate.berty.community/
-
-We encourage you to read our dedicated [blog post](https://berty.tech/blog/berty-translation/)
+We encourage you to open a new PR on the [berty](https://github.com/berty/berty) repository.
+All the translations are stored in the [/i18n/locale](https://github.com/berty/berty/tree/master/js/packages/i18n/locale) folder.
```

Comment:
This is for contributing translations to the Berty app. Do you want to add links for contributing translations to the Berty web site? I ask because we do already have translations: https://berty.tech/pt

Reply:
I've included a simple website mention in this commit [chore: translation instruction for website](https://github.com/berty/www.berty.tech/pull/411/commits/d65e52f792f1fdbfe18916db20aedffa2b4647fb)